src/components/Course/ChapterStrip.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <div>
+    <b-card>
+      <b-card v-b-toggle="'collapse-' +chapter._id">
+        {{ chapter.title }}
+      </b-card>
+      <b-collapse
+        :id="'collapse-'+chapter._id"
+        class="mt-2"
+      >
+        <b-card
+          v-for="(section, index) in chapter.sections"
+          :key="index"
+        >
+          <div>Chapter {{ index+1 }}: {{ section.title }}</div>
```

Comment:
use field chapterNo from api call. do not depend on the index received from db. Also this looks like a for loop for sections but showing Chapter number